docs/geps/gep-00.rst

```diff
@@ -178,9 +178,9 @@ more feedback and looking for opportunities to compromise.
 
 If the final comment period passes without any substantive objections, then the GEP can
 officially be marked ``Accepted``. You should send a follow-up message notifying the
-community (celebratory emoji optional but encouraged 🎉✨), and then update the GEP by
-setting its ``:Status:`` to ``Accepted``, and its ``:Resolution:`` header to a link to
-your follow-up message.
+community (celebratory emoji optional but encouraged 🎉✨), and then update the GEP
+by setting its ``:Status:`` to ``Accepted``, and its ``:Resolution:`` header to a link
```

Comment:
Can you revert these, please? Or did they have any meaning?

Reply:
Yes, because the emoticons are one character for all editors but apparently more for doc8 which causes an max line length exceeded error.

Reply:
Fine, but why does it care if the file does not change otherwise in this PR ?

Reply:
Formerly, you had to use `max-line-length=89` with `doc8` because it was treated like 88 characters. Since PyCQA took over the project, they fixed the bug. Then, you created the gep. Now, I changed the setting to 88 characters. That is why the error occurs now.

Reply:
Thanks for the explanation!